jumanji/environments/swarms/search_and_rescue/utils.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import chex
+import jax
+import jax.numpy as jnp
+from esquilax.utils import shortest_vector
+
+from jumanji.environments.swarms.common.types import AgentState
+from jumanji.environments.swarms.search_and_rescue.types import TargetState
+
+
+def has_been_found(
```

Comment:
These method names are very similar and quite confusing.

I'd call one `target_in_range` and the other `reward_if_found_target`